File: Applications/C++ & C# programming languague/Lab5/Lab5Tema/Lab5tema/Pro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab5tema
{
    public class Pro : Active
    {
        public string sponsor;
        public int earnings;

        public double get_earnings()
        {
            return earnings;
        }

        //Lab5
        public void display_years_activity()
        {
            Console.WriteLine("Display years of activity: " + years_activity + " (called by Pro Class)");
        }

        public override void display_suspended()
        {
            Console.WriteLine("Display days as suspended: " + suspended + " (called by Pro Class)");
        }
    }
}

File: Applications/C++ & C# programming languague/Lab4/Lab4Tema - metoda2/Lab1Tema/Pro.cpp
#include "Pro.h"
#include <iostream>

using namespace std;

double Pro::get_earnings()
{
	return earnings;
}

void Pro::Pro_E_access()			//Lab3ex5
{
	consume.display_hydrated();
}

//Lab4

void Pro::display_years_activity()
{
	cout << "Display years of activity: " << years_activity <<" (called by Pro Class)" << endl;		//Lab4ex1
}

void Pro::display_suspended()
{
	cout << "Display days as suspended: " << suspended << " (called by Pro Class)" << endl;		//Lab4ex2
}